client/src/common/components/Members/components/MembersForm.jsx

```diff
@@ -81,7 +86,7 @@ class MembersForm extends PureComponent {
           className={classNames('primary-button', {
             'primary-button--disabled': disabled
           })}
-          disabled={disabled}
+          disabled={disabled || pending || isEmpty}
```

Comment:
create const above

`const isButtonDisabled = disabled || pending || isEmpty; `

and use it here